alignak_module_nsca/etc/arbiter/modules/mod-nsca.cfg

```diff
@@ -6,6 +6,9 @@ define module {
     module_types            passive
     python_name             alignak_module_nsca
 
+    # Module log level
```

Comment:
Can you ad in the comment the different values available? `INFO`, `DEBUG`...

Reply:
Done